javasphinx/compiler.py

```diff
@@ -339,4 +339,9 @@ def compile(self, ast):
             document = self.compile_type_document(import_block, package, name, declaration)
             documents[full_name] = (package, name, document.build())
 
+
         return documents
+
+    def compile_docblock(self, documented):
+        ''' Compiles a single, standalone docblock. '''
+        return self.__output_doc(documented).build()
```

Comment:
In fact it works. I tried that approach but I was getting an error in Visual Studio Code, complaining that such a "package" attribute didn't exist. But now I understood why: the attributes are created dynamically when the java file is read.